lamindb/schema/__init__.py

```diff
@@ -44,7 +44,7 @@
 See `lnschema-wetlab <https://lamin.ai/docs/lnschema-wetlab/api>`__.
 """
 
-from ._core import draw, list_entities
+from ._core import draw, get_db_metadata_as_dict, list_entities
```

Comment:
I'd prefer to no re-export `get_db_metadata_as_dict` for the typical user. To me, that's a dev function that is either accessed through `schema._core` or through `.dev`.

Reply:
It's important that the auto-lookup for `ln.schema.*` displays a concise list of user-centered functionality.

Reply:
Yes totally agree